From ae120d987d2ba906d8d6e44d138e91cc63430fb2 Mon Sep 17 00:00:00 2001
Message-ID: <ae120d987d2ba906d8d6e44d138e91cc63430fb2.1705743627.git.sharlatanus@gmail.com>
In-Reply-To: <cover.1705743627.git.sharlatanus@gmail.com>
References: <cover.1705743627.git.sharlatanus@gmail.com>
From: Sharlatan Hellseher <sharlatanus@gmail.com>
Date: Tue, 16 Jan 2024 22:48:34 +0000
Subject: [PATCH 06/38] gnu: go-github-com-shadowsocks-go-shadowsocks2: Move to
 (gnu packages golang-crypto).

* gnu/packages/golang.scm (go-github-com-shadowsocks-go-shadowsocks2):
Move from here ...
* gnu/packages/golang-crypto.scm: ... to here.

Change-Id: Iae18376cae074b6ffb2d5b1b27b6161f4712dfdb
---
 gnu/packages/golang-crypto.scm | 29 +++++++++++++++++++++++++++++
 gnu/packages/golang.scm        | 27 ---------------------------
 2 files changed, 29 insertions(+), 27 deletions(-)

diff --git a/gnu/packages/golang-crypto.scm b/gnu/packages/golang-crypto.scm
index 2c9605b8cf..859d490787 100644
--- a/gnu/packages/golang-crypto.scm
+++ b/gnu/packages/golang-crypto.scm
@@ -120,6 +120,35 @@ (define-public go-github-com-riobard-go-bloom
       (description "Go-Bloom implements bloom filter using double hashing.")
       (license license:asl2.0))))
 
+(define-public go-github-com-shadowsocks-go-shadowsocks2
+  (package
+    (name "go-github-com-shadowsocks-go-shadowsocks2")
+    ;; Version > 0.1.3 requires go-toolchain v1.16.
+    (version "0.1.3")
+    (source
+     (origin
+       (method git-fetch)
+       (uri (git-reference
+             (url "https://github.com/shadowsocks/go-shadowsocks2")
+             (commit (string-append "v" version))))
+       (file-name (git-file-name name version))
+       (sha256
+        (base32 "1wzy3ml4ld83iawcl6p313bskzs6zjhz8vlg8kpwgn71cnbv4pvi"))))
+    (build-system go-build-system)
+    (arguments
+     `(#:import-path "github.com/shadowsocks/go-shadowsocks2"))
+    (propagated-inputs
+     (list go-github-com-riobard-go-bloom
+           go-golang-org-x-crypto
+           go-golang-org-x-net
+           go-golang-org-x-sys
+           go-golang-org-x-text))
+    (home-page "https://github.com/shadowsocks/go-shadowsocks2")
+    (synopsis "Shadowsocks tunnel proxy")
+    (description "Go-ShadowSocks is a Go implementation of the Shadowsocks
+tunnel proxy protocol.")
+    (license license:asl2.0)))
+
 ;;;
 ;;; Avoid adding new packages to the end of this file. To reduce the chances
 ;;; of a merge conflict, place them above by existing packages with similar
diff --git a/gnu/packages/golang.scm b/gnu/packages/golang.scm
index 2bf6b0a4a9..a47345cc1e 100644
--- a/gnu/packages/golang.scm
+++ b/gnu/packages/golang.scm
@@ -1733,33 +1733,6 @@ (define-public go-github-com-rfjakob-eme
 mode developed by Halevi and Rogaway.")
     (license license:expat)))
 
-(define-public go-github-com-shadowsocks-go-shadowsocks2
-  (package
-    (name "go-github-com-shadowsocks-go-shadowsocks2")
-    ;; Version > 0.1.3 requires go-toolchain v1.16.
-    (version "0.1.3")
-    (source
-     (origin
-       (method git-fetch)
-       (uri
-        (git-reference
-         (url "https://github.com/shadowsocks/go-shadowsocks2")
-         (commit (string-append "v" version))))
-       (file-name (git-file-name name version))
-       (sha256
-        (base32 "1wzy3ml4ld83iawcl6p313bskzs6zjhz8vlg8kpwgn71cnbv4pvi"))))
-    (build-system go-build-system)
-    (arguments
-     `(#:import-path "github.com/shadowsocks/go-shadowsocks2"))
-    (propagated-inputs
-     (list go-github-com-riobard-go-bloom go-golang-org-x-crypto
-           go-golang-org-x-net go-golang-org-x-sys go-golang-org-x-text))
-    (home-page "https://github.com/shadowsocks/go-shadowsocks2")
-    (synopsis "Shadowsocks tunnel proxy")
-    (description "Go-ShadowSocks is a Go implementation of the Shadowsocks tunnel
-proxy protocol.")
-    (license license:asl2.0)))
-
 (define-public go-github-com-schachmat-ingo
   (package
     (name "go-github-com-schachmat-ingo")
-- 
2.41.0

